### DISCUSSION TITLE
v0.20.1 (BREAKING CHANGES, ACTION REQUIRED)

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1474] [Releases]
> [!WARNING]  
> `v0.20.x` has breaking changes and requires action. Read the original release notes here: https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.0

## What's Changed
* [Bug] Default display timezone to utc by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1471
* [Bug] Display timezone should be used for scheduled tests by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1472
* Release v0.20.1 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1473

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.20.0...v0.20.1

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.20.1'>v0.20.1 (BREAKING CHANGES, ACTION REQUIRED)</a>.</em>

**Comment by masterwishx** [comment]
Added all new env to docker containers in Unraid and vps ...

Why is config removed from App if I may ask?

Time format is not available anymore?

**Reply by alexjustesen** [reply]
You can pass `DATETIME_FORMAT` as an environment variable to modify how dates are shown, also this https://github.com/alexjustesen/speedtest-tracker/discussions/1470#discussioncomment-9701663 is why. In short it was causing me a lot of headaches.

**Reply by masterwishx** [reply]
> You can pass `DATETIME_FORMAT` as an environment variable to modify how dates are shown

sorry i forgot `DATETIME_FORMAT` only work for Results page but not for Dashboard page?

**Reply by masterwishx** [reply]
OK, updated to lasted version with the fix
